pkgs/applications/system/mgmt/default.nix

```diff
@@ -0,0 +1,73 @@
+{ augeas
+, bash
+, buildGoModule
+, fetchFromGitHub
+, gotools
+, lib
+, libvirt
+, libxml2
+, nex
+, pkg-config
+, ragel
+}:
+buildGoModule rec {
+  pname = "mgmt";
+  version = "unstable-2022-10-24";
+
+  src = fetchFromGitHub {
+    owner = "purpleidea";
+    repo = pname;
+    rev = "d8820fa1855668d9e0f7a7829d9dd0d122b2c5a9";
+    hash = "sha256-jurZvEtiaTjWeDkmCJDIFlTzR5EVglfoDxkFgOilo8s=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile --replace "/usr/bin/env " ""
+    substituteInPlace examples/dhcp_client/Makefile --replace "/usr/bin/env " ""
+    substituteInPlace lang/Makefile --replace "/usr/bin/env " ""
+    substituteInPlace lang/fuzz/Makefile --replace "/usr/bin/env " ""
+    substituteInPlace lang/types/Makefile \
+      --replace "/usr/bin/env " "" \
+      --replace "unset GOCACHE && " ""
+    substituteInPlace misc/mkosi/Makefile --replace "/usr/bin/env " ""
+    substituteInPlace test/Makefile --replace "/usr/bin/env " ""
+    patchShebangs misc/header.sh
```

Comment:
```suggestion
    for file in Makefile **/Makefile; do
      substituteInPlace $file --replace "/usr/bin/env " ""
    done
    
    substituteInPlace lang/types/Makefile \
      --replace "unset GOCACHE && " ""
    patchShebangs misc/header.sh
```

Reply:
this does not actually work, `**/Makefile` only matches `Makefile`